packages/browsers/src/browsers/chrome.ts

```diff
@@ -57,26 +57,18 @@ export function resolveDownloadUrl(
   )}.zip`;
 }
 
-export function executablePath(
+export function relativeExecutablePath(
   platform: BrowserPlatform,
-  revision: string,
-  basePath = ''
+  _revision: string
```

Comment:
Why keep here?

Reply:
The main reason is that the binary path might change based on the revision which is currently not used but if we expose it on the API, it might be useful in the future.